Interface.C

```diff
@@ -275,6 +280,75 @@ void preciceAdapter::Interface::configureMesh(const fvMesh& mesh)
             }
         }
     }
+
+    else if (locationType_ == LocationType::volumeCenters)
+    {
+        // The volume coupling implementation is based on the coupling with faceCenters
+        // as the volume coupling was tested and developed on a previous development where faceNodes
+        // did not work properly
```

Comment:
I am not sure I understand this comment. Do you mean that the branch that we started from did not yet have `faceNodes` implemented or complete?

In any case, we are assuming cell-based FVM, which is the most common approach (there is also nodes-based FVM, but this is way out of scope of the complete adapter at the moment, as only some extensions of OpenFOAM implement it). As such, all solvers store all fields (with some exceptions) in the centers of the cells.

Independent on what the development was based upon, why does it matter?

Reply:
This is a comment from Prasad's original fork. It might have been relevant at the time, but if it isn't anymore I can remove it (or only leave the first line)

Reply:
I think we can remove this.